### PR TITLE
docs(highlights): add highlights for 0.26.0

### PR DESCRIPTION
**Pull request by orhun**

### CODE REVIEW

**Comment by kdheepak** [comment]
This looks great! Thanks for writing it up! 

I think we should mention the `Fill` constraint in a separate section in addition to flex mode changes. 

I can update the flex section with some more details and link to another page that has even more information, if you want to merge as is.

**Comment by orhun** [comment]
Feel free to push to this PR if you want to make changes or we can just merge it! (But we probably can get the opinions of Josh and Valentin before that.)

File: src/content/docs/highlights/v0.26.md [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+    Constraint::Min(5),
+    Constraint::Length(10),
+]);
+```
+
+---
+
+## Layout: Accept Constraints 📐
+
+The Layout constructors now accept any type that implements Into<Constraint> instead of just
```
Comment (joshka) [inline src/content/docs/highlights/v0.26.md]: Backticks needed here

File: src/content/docs/highlights/v0.26.md [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+```
+
+Check out the [pull request](https://github.com/ratatui-org/ratatui/issues/881) for the motivation
+behind this feature and more information.
+
+---
+
+## Rect: `contains` 🔲
+
+If you want to perform hit tests, this new method is for you. (e.g. did the user clicked in an area)
```
Comment (joshka) [inline src/content/docs/highlights/v0.26.md]: click

**Review by joshka — APPROVED** [review]
I'd say the most important changes worth putting up top are:
- Flex
- ref widget

**Comment by kdheepak** [comment]
I was trying to add this gif but it doesn't seem to be rendering locally for me. I've just pushed it using `git lfs`:

constraints

**Pull request merged**